3rdparty/.gitignore

```diff
@@ -0,0 +1 @@
+Makefile
```

Comment:
Here this should be `/Makefile`, since other third-party contributions may include files called `Makefile`.

Reply:
a6ced7d39809af94ab5ce3d21e24f0438a54e576